### ISSUE TITLE
Variable p (of Scene.PropStruct) remains null when rendering Prop

### ISSUE THREAD

**Issue by matthewd673** [neon2d/neon2d#44]
If you attempt to create a Prop without specifying a width and height (leaving it default) it will run into a _NullReferenceException_ when trying to render in Game.cs

**Comment by matthewd673** [comment]
EDIT: This seems to happen either way, it could be that renderlist[i] is null but that should never happen since its in an ArrayList not a typical array

**Comment by DontBelieveMe** [comment]
This seems to work for me. Can you show me the code?

**Comment by matthewd673** [comment]
Code in my project: `computer = new Prop(new Bitmap(Environment.CurrentDirectory + @"\computer.png"), 24, 24);`

Code from Game.cs:

```
else if(scene.renderlist[i].GetType() == typeof(Scene.PropStruct))
                {
                    //its a prop
                    Scene.PropStruct placeholder = (Scene.PropStruct)scene.renderlist[i];

                    g.DrawImage(placeholder.p.propSource, new Rectangle(placeholder.x, placeholder.y, placeholder.p.propWidth, placeholder.p.propHeight));
                }
```

Looking closer into it, it appears p (part of Scene.PropStruct) = null, the other values are fine

**Comment by matthewd673** [comment]
Looking through the Scene.cs and Prop.cs code there doesn't seem to be anything wrong with it (variables not getting set, etc.)

**Comment by DontBelieveMe** [comment]
Is computer.png in the res folder?

**Comment by matthewd673** [comment]
Yeah. I don't think the issue is with the image file, as the entire Prop is seen as null, not just the Bitmap

**Comment by DontBelieveMe** [comment]
Can you upload the code to a branch? I think I may know what the problem is, but I need to test it for definite.

**Comment by matthewd673** [comment]
I thought the code between my local disk and the repo was synced. Since you aren't having problems I'll just wipe my local repo and clone a fresh one from GitHub. Thats what I do when my changes get too messy.

**Comment by DontBelieveMe** [comment]
Has this fixed your problem?

**Comment by matthewd673** [comment]
@DontBelieveMe The problem persists even when I wipe my local repo and clone a fresh one

**Comment by DontBelieveMe** [comment]
If the image is in the res folder then the path should be 
`Environment.CurrentDirectory + @"\..\..\computer.png"`

because the `res` folder is two folders above the `bin` folder (which is `Enviroment.CurrentDirectory`).

**Comment by matthewd673** [comment]
Thanks for your help

**Comment by DontBelieveMe** [comment]
Did that work?
[EDIT:] If that worked then this Issue can probably be closed.
